src/completed.js

```diff
@@ -0,0 +1,34 @@
+const todoes = JSON.parse(localStorage.getItem('todoes'));
+
+const DeleteChecked = () => {
+  const temp = todoes.filter(todo => todo.completed === false);
+
+  localStorage.setItem('todoes', JSON.stringify(temp));
+  const newTodoes = JSON.parse(localStorage.getItem('todoes'));
+
+  for (let i = 0; i < newTodoes.length; i += 1) {
+    newTodoes[i].index = i + 1;
+  }
+  localStorage.setItem('todoes', JSON.stringify(newTodoes));
```

Comment:
I notice you are using this in many parts of your code, maybe you can create a function save() and put it in all the parts of your code that need it.